pkg/sentry/fsimpl/devpts/line_discipline.go

```diff
@@ -212,9 +212,15 @@ func (l *lineDiscipline) inputQueueRead(ctx context.Context, dst usermem.IOSeque
 			l.replicaWaiter.Notify(waiter.ReadableEvents)
 		}
 		return n, nil
-	} else if notifyEcho {
-		l.masterWaiter.Notify(waiter.ReadableEvents)
+	} else {
```

Comment:
Nit: `else` is not needed in this case, since the preceeding `if` block returns unconditionally.